src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/BasePdfTest.java

```diff
@@ -40,12 +45,8 @@ private InputStream retrievePdf(String pdfName, String externalId) {
             .asInputStream();
     }
 
-    protected InputStream retrieveCCJPdf(String externalId) {
-        return RestAssured
-            .given()
-            .header(HttpHeaders.AUTHORIZATION, user.getAuthorisation())
-            .get("/documents/ccj/" + externalId)
-            .asInputStream();
+    protected InputStream retrieveCCJPdf(Claim claim) {
+        return new ByteArrayInputStream(countyCourtJudgmentPdfService.createPdf(claim));
```

Comment:
Any particular reason we are using `countyCourtJudgmentPdfService` to create Pdf, as in all functional test we are trying to call individual apis.

Reply:
@ckalista @srineelesh Any reason we have polluted AAT tests with claim store services as are trying to test and use just end points here.